includes/Core/Modules/Modules.php

```diff
@@ -468,6 +470,19 @@ public function activate_module( $slug ) {
 		return true;
 	}
 
+	/**
+	 * Checks whether the module identified by the given slug is enabled by the option.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param string $slug Unique module slug.
+	 * @return bool True if module has been manually enabled, false otherwise.
+	 */
+	public function manually_enabled( $slug ) {
```

Comment:
Let's not make this public - it's not a method intended for others to call, and it doesn't need to be public for the code here either.

Reply:
https://github.com/google/site-kit-wp/pull/4305/commits/96b968af5c7efc192fb6ddd4c5e3269ad0f1debe